test/necb/unit_tests/expected_results/necb2011_compliance_unitary_efficiencies_expected_results.csv

```diff
@@ -1,9 +1,9 @@
 Heating Type,Min Capacity (Btu per hr),Max Capacity (Btu per hr),Seasonal Energy Efficiency Ratio (SEER),Energy Efficiency Ratio (EER)
-Electric Resistance,0,65000,14.0,
-Electric Resistance,65001,249999,,9.7
-Electric Resistance,250000,759999,,8.39
-Electric Resistance,760000,9999999999,,8.12
-All Other,0,65000,14.0,
-All Other,65001,249999,,9.7
-All Other,250000,759999,,8.21
-All Other,760000,9999999999,,7.94
\ No newline at end of file
+Electric Resistance,0,65000,17.98,
```

Comment:
These expected test results were, I think, incorrect. They were based on a COP to SEER conversion that first converted the COP to EER_no_fan and used this value to convert the SEER. I believe that the correct approach is that convert the COP to EER and then convert it to a SEER value. That is what the updated function does, I'm just not sure why it passed #1676.